cpp/src/arrow/csv/options.h

```diff
@@ -163,6 +163,9 @@ struct ARROW_EXPORT ReadOptions {
   /// If false, column names will be read from the first CSV row after `skip_rows`.
   bool autogenerate_column_names = false;
 
+  /// Character encoding used
+  std::string encoding = "UTF-8";
```

Comment:
```suggestion
  /// Character encoding used. Only "UTF-8" is supported.
  std::string encoding = "UTF-8";
```

Reply:
nit: maybe a formal constant for the encoding?

Reply:
Done